rcl/include/rcl/types.h

```diff
@@ -126,4 +126,5 @@ typedef rmw_ret_t rcl_ret_t;
 /// typedef for rmw_serialized_message_t;
 typedef rmw_serialized_message_t rcl_serialized_message_t;
 
+
```

Comment:
nitpick: minimize vertical whitespace (https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace)

also unrelated to changes in this pr